cli.php

```diff
@@ -35,7 +37,10 @@
 // Create the CLI command instance and add to WP CLI.
 $plugin_command = new Plugin_Check_Command( $context );
 WP_CLI::add_command( 'plugin', $plugin_command );
-
+$plugin_list_check_categories_command = new Plugin_List_Check_Categories_Command( $context );
+WP_CLI::add_command( 'plugin', $plugin_list_check_categories_command );
+$plugin_list_checks_command = new Plugin_List_Checks_Command( $context );
+WP_CLI::add_command( 'plugin', $plugin_list_checks_command );
```

Comment:
No need for separate classes, it makes the code more complicated than it needs to be. When adding a class as WP-CLI command, all public methods become commands on the same namespace.

Especially since the classes share some commonalities, we should add the new commands logic to the existing `Plugin_Check_Command` class instead. This will avoid some duplicate code.